object.go

```diff
@@ -1944,7 +1944,7 @@ func (o *baseObject) estimateMemUsage(ctx *MemUsageContext) (uint64, error) {
 		inc, err := v.MemUsage(ctx)
 		samplesVisited += 1
 		total += inc
-		total += uint64(len(k))
+		total += uint64(len(k)) + SizeString
 		averageMemUsage = float32(total) / float32(samplesVisited)
```

Comment:
[opt] I think you can avoid doing this calculation on every loop and only do it on the return. Sample function to keep it consistent.
```go
func roughMemUsage(sampleTotal, numSamples uint64, totalProps int) uint64 {
  return uint64(math.Ceil((float32(sampleTotal) / float32(numSamples)) * float32(totalProps)))
}
```